state-chain/pallets/cf-elections/src/electoral_systems/blockchain/delta_based_ingress.rs

```diff
@@ -157,7 +160,10 @@ where
 			<Self::Vote as VoteStorage>::Vote,
 		),
 	) -> bool {
-		current_partial_vote != proposed_partial_vote
+		proposed_partial_vote.into_iter().any(|(account, new_balance)| {
+			new_balance.amount !=
+				current_partial_vote.get(&account).map(|total| total.amount).unwrap_or_default()
```

Comment:
As discussed, we need to think about what this means for closing of channels. We need a vote to trigger channel closure, but this will filter out votes unless there is a balance update.